envs/env-py39.yml

```diff
@@ -65,7 +65,7 @@ dependencies:
   - memory_profiler
   - mendeleev
   - modestimage
-  - mxtools >=0.0.8
+  - mxtools >=1.0.1
```

Comment:
I think the env name on top of this file will need to be updated too.

Reply:
done in https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/11/commits/202e38e7b0e8034e2585d3b582776f43aca9872b